README.md

```diff
@@ -265,15 +265,33 @@ func (n *NameDotName) UnmarshalText(b []byte) error {
 	return nil
 }
 
+// optional, implement in case you want to use defaults
```

Comment:
perhaps change to "optional: implement in case you want to display a default value in the usage string"